tests/wpt/metadata/MANIFEST.json

```diff
@@ -37215,6 +37215,12 @@
     "deleted_reftests": {},
     "items": {
       "testharness": {
+        "dom/nodes/NodeList-Iterable.html": [
+          {
+            "path": "dom/nodes/NodeList-Iterable.html",
+            "url": "/dom/nodes/NodeList-Iterable.html"
+          }
+        ],
```

Comment:
@izgzhen I'm not bold enough to modify manifest file :p
I was using `./mach create-wpt tests/.../foo.html`, as recommended [here](https://github.com/servo/servo/tree/master/tests/wpt#writing-new-tests)
I'll check out your advice too.